integration/networking/bridge_linux_test.go

```diff
@@ -615,6 +615,61 @@ func TestDisableIPv6Addrs(t *testing.T) {
 	}
 }
 
+// Check that a container in a network with IPv4 disabled doesn't get
+// IPv4 addresses.
+func TestDisableIPv4(t *testing.T) {
+	ctx := setupTest(t)
+	d := daemon.New(t, daemon.WithExperimental())
```

Comment:
It'll be needed until it's removed by:
  - https://github.com/moby/moby/pull/48809 - stacked on top of this PR and others so, just the last commit.

The [first "IPv6 only" PR](https://github.com/moby/moby/pull/48271) added `EnableIPv4` to the API, but it didn't actually do anything, I'm building up the functionality behind it over about ten following PRs. So, in that first PR, I thought it was best to require `--experimental` for `EnableIPv4: false` so that, if we ship from master before the rest is all merged, there will at-least be a hint that IPv6 only is incomplete.